tests/acceptance/features/apiGraph/assignRole.feature

```diff
@@ -35,7 +35,7 @@ Feature: assign role
 
   Scenario Outline: assign role to the user with setting api and list role with graph api
     Given user "Alice" has been created with default attributes and without skeleton files
-    And the administrator has assigned the role "<userRole>" to user "Alice" using the Graph API
+    And the administrator has given "Alice" the role "<userRole>" using the settings api
```

Comment:
Why change this to setting api?

Reply:
I read title: `Scenario Outline: assign role to the user with setting api and list role with graph api`